features/js-self-profiling.yml

```
@@ -1,7 +1,7 @@
-draft_date: 2024-07-11
 name: JS Self-Profiling API
```

In "JavaScript modules" the JS is expanded, and we avoid API in many (but not all) cases. Would one of the following names work?

```suggestion
name: JavaScript self-profiling
```

```suggestion
name: Self-profiling
```

```suggestion
name: Profiler
```

For the identifier, `profiler.yml` is a pretty safe choice since it matches the name of the constructor, regardless of the name.

---

👍 to not using abbreviations in names, in any case. Strangely, I think we should avoid "JavaScript" here because it implies that it's a JS API (like `Array`) and not a Web API.

And I suspect that "Self-profiling" is the right thing today, but in the I bet we'll end up renaming this to "Profiler" once it's well-established.